src/components/SignUp/Landing.tsx

```diff
@@ -202,12 +237,74 @@ const Landing: React.FC<{
 };
 
 const StyledCard = styled(Col)`
+  width: 273px;
+  --cardHeight: 353.23px;
```

Comment:
What we could do is create a new object in styleVars for `sizes`, so we can do this the JS way and use a JS property from the sizes object for `cardHeight` instead. Or, if there's nowhere else in the app we'd ever need this height, we could just create a JS variable right about StyleCard that's:
```
const cardHeight = 353;
```

Notice the lack of a decimal, too: I don't think we have to be so specific to the Figma doc where we end up using decimals haha.